docs/api/version-history.md

```diff
@@ -38,6 +38,11 @@ keywords: "API, Docker, rcli, REST, documentation"
   specifications directories. The use of the applied setting requires the daemon
   to have expermental enabled, and for non-experimental daemons an empty list is
   always returned.
+* `POST /containers/create` and `POST /networks/{id}/connect` will now catch
+  validation errors that were previously only returned during `POST /containers/{id}/start`.
+  These endpoints will also return the full set of validation errors they find,
+  instead of returning only the first one.
+  Note that this change is _unversioned_ and applies to all API versions.
```

Comment:
I want to double-check we agree to make this change unversioned before editing Swagger files for older versions.

Reply:
Discussed during maintainers' call, and that's okay.